_posts/2022-07-01-So-You-Want-To-eWaterCycle.md

```diff
@@ -0,0 +1,75 @@
+
+### Models and datasets
+
+All of the above ways of working with eWaterCycle will give you access to the models and datasets that ‘ship’ with the package. On our demonstration machine we have added a few more datasets, see [https://www.ewatercycle.org/demo](https://www.ewatercycle.org/demo) for details. eWaterCycle is a work in progress and we keep adding models and datasets as we move forward. 
+
+Currently adding one's own model requires quite a few steps as described [in the documentation](https://ewatercycle.readthedocs.io/en/latest/adding_models.html). Simplifying this process is one of the things we will be working on in the future (see below). 
+
+### Future options: what are we working on?
+
+Our goal is to provide easy access to eWaterCycle to as many researchers and students as possible. To achieve this we are working on the following options:
+
+#### eWaterCycle for education: Open Educational Resources
+
+Through a grant of the Dutch ministry of Education and Science we will be importing all educational material on computational hydrological modeling being taught at Dutch universities into eWaterCycle and make those available as ‘Open Educational Resources’ for anyone to use. We in this case is a collaboration between Delft University of Technology, Wageningen University and Research, University Twente and the Netherlands eScience center. 
+
+#### eWaterCycle at Delft University of Technology
+
+Of course we are using eWaterCycle for our own hydrology research. Recent examples (using older versions of the platform than currently available!) include 
+
+* Coupling a global glacier model to a global hydrological model prevents underestimation of glacier runoff  ([Wiersma 2022, in Review in HESS](https://doi.org/10.5194/egusphere-2022-106))
+* Large-sample assessment of spatial scaling effects of the distributed wflow_sbm hydrological model shows that finer spatial resolution does not necessarily lead to better streamflow estimates ([Aerts 2021, in Review in HESS](https://doi.org/10.5194/hess-2021-605))  
+
+#### eWaterCycle far future
+We are currently looking for funds and partnerships to keep on expanding both access to eWaterCycle as well as expanding the features of, and models and data available through, eWaterCycle. If you want to collaborate, or host eWaterCycle for your community or for the entire world, please contact us at [questions@ewatercycle.org](mailto:questions@ewatercycle.org). 
```

Comment:
Should we add that adding good standards and support for easily exchanging and manipulating parameter sets is high on our wishlist?

Reply:
added a short sentence on datasets, did not want to go to deep into it, not wanting to promise things we might not get to. Agree?